cfme/tests/intelligence/test_chargeback_assignments.py

```diff
@@ -1,137 +1,138 @@
 # -*- coding: utf-8 -*-
-import cfme.intelligence.chargeback.assignments as cb
 import pytest
 import random
 
-from cfme.intelligence.chargeback.assignments import AssignmentsView
 from cfme import test_requirements
-from cfme.utils.appliance.implementations.ui import navigate_to
+from cfme.utils.appliance.implementations.ui import navigate_to, navigator
 
 pytestmark = [
     pytest.mark.tier(3),
     test_requirements.chargeback
 ]
 
 
+@pytest.fixture
+def assigns_collection(appliance):
```

Comment:
This doesn't need to be a fixture, provides no value - this collection object can just be referenced in the test directly through its existing appliance fixture.

Reply:
Ah, I see, on looking up the use of pytest fixtures, I think it works in a manner similar to that of Mockbeans in Mockito(java). So, I am able to understand why this need not be a fixture